test/Elastic.Apm.AspNetCore.Tests/CaptureUserTest.cs

```diff
@@ -0,0 +1,137 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Reflection;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Elastic.Apm.Api;
+using Elastic.Apm.Model.Payload;
+using Elastic.Apm.Tests.Mocks;
+using FluentAssertions;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.DependencyInjection;
+using SampleAspNetCoreApp;
+using Xunit;
+
+namespace Elastic.Apm.AspNetCore.Tests
+{
+
+	/// <summary>
+	/// The ASP.NET Core auto instrumentation is able to capture logged in users.
+	/// This class tests that feature.
+	/// </summary>
+	[Collection("DiagnosticListenerTest")]
+	public class CaptureUserTest : IDisposable
+	{
+		private readonly MockPayloadSender _payloadSender = new MockPayloadSender();
+		private ApmAgent _agent;
+
+		private void SetUpSud()
```

Comment:
Did you mean `SetUpSut` (SUT - System Under Test)?

Reply:
Yes, fixed. Thx!